src/polymer/polymer-element.ts

```diff
@@ -183,7 +183,15 @@ export class ScannedPolymerElement extends ScannedElement implements
     addMethod(this, method);
   }
 
+  applyDescriptionAsJsdoc() {
+    this.jsdoc = jsdoc.parseJsdoc(this.description);
+    this.description = this.jsdoc.description;
+  }
+
   resolve(document: Document): PolymerElement {
+    if (!this.jsdoc) {
+      this.applyDescriptionAsJsdoc();
+    }
     this.applyJsdocDemoTags(document.url);
```

Comment:
In my prior merged PR which added the `applyJsdocDemoTags()` method I had not realized that `resolve()` was intended to be idempotent and also not modify the scanned element.  Am working on update to address that error as well.